sig/datadog/core/telemetry/event/app_started.rbs

```diff
@@ -3,12 +3,16 @@ module Datadog
     module Telemetry
       module Event
         class AppStarted < Base
+          @configuration: Array[Hash[Symbol, untyped]]
```

Comment:
This should be defined outside as a type alias instead. Same applies to next 2 instance variables